pysrc/scripts/sampling.py

```diff
@@ -185,8 +192,8 @@ def run_experiments(config: GlobalConfiguration) -> str:
         ))
 
     #TODO change to clever sampling schema change the cartesian product
-    file_combinations_totest = get_File_Combinations(samples, config)
-    for file_combination_setup in file_combinations_totest:
+    file_combinations_to_test = get_file_combinations(samples, config)
+    for file_combination_setup in file_combinations_to_test:
```

Comment:
I don't like the type error there. I think these should be tuples instead of lists, but this would require more changes in the whole project, so not for now. But we should keep it in mind in case we ever come across a situation where this might have been the problem.